brax/training/networks.py

```diff
@@ -94,7 +94,8 @@ def make_policy_network(
     hidden_layer_sizes: Sequence[int] = (256, 256),
     activation: ActivationFn = linen.relu,
     kernel_init: Initializer = jax.nn.initializers.lecun_uniform(),
-    layer_norm: bool = False) -> FeedForwardNetwork:
+    layer_norm: bool = False,
+    state_key: str = 'state') -> FeedForwardNetwork:
```

Comment:
maybe this should be `obs_key`? Either that or add a docstring, it's a fairly unintuitive field as opposed to the others